core/arxiv/submission/domain/submission.py

```diff
@@ -79,6 +79,17 @@ class SubmissionCompiled:
     size: int
 
 
+@dataclass
+class Compilation:
+    """Represents a submission compilation."""
+
+    task_id: str
+    source_etag: str
+    format: str
+    start_time: datetime = field(default_factory=get_tzaware_utc_now)
+    status: str
```

Comment:
Consider adding a default status? Also, what values are acceptable? (consider an Enum type)

Reply:
Made these mirror the status in compiler.domain: https://github.com/arXiv/arxiv-compiler/blob/develop/compiler/domain.py#L33